src/gccrs/mod.rs

```diff
@@ -73,4 +77,40 @@ impl Gccrs {
         Gccrs::dump_config()?;
         GccrsConfig::display()
     }
+
+    fn spawn_with_args(args: &Vec<String>) -> Result<ExitStatus> {
```

Comment:
Clippy would complain here, that you use `&Vec<_>` over `&[_]`. Also in the other functions below.